src/components/TableCell/TableCell.test.tsx

```diff
@@ -1,42 +1,44 @@
-import { shallow } from "enzyme";
+/* eslint-disable testing-library/no-node-access */
```

Comment:
This test is interacting with the nodes directly as the `<td />` will never appear in isolation (outside of the table) and unit-style test makes more sense in this case.